subsystem/cluster_test.go

```diff
@@ -781,6 +781,76 @@ var _ = Describe("cluster install", func() {
 
 		})
 
+		It("[only_k8s]install_cluster install command failed", func() {
+			By("Installing cluster till finalize")
+			c := installCluster(clusterID)
+			Expect(swag.StringValue(c.Status)).Should(Equal("installing"))
+			Expect(swag.StringValue(c.StatusInfo)).Should(Equal("Installation in progress"))
+			Expect(len(c.Hosts)).Should(Equal(4))
+			for _, host := range c.Hosts {
+				Expect(swag.StringValue(host.Status)).Should(Equal("installing"))
+			}
+
+			for _, host := range c.Hosts {
+				updateProgress(*host.ID, clusterID, models.HostStageDone)
+			}
+			// post failure to execute the install command
+			_, err := agentBMClient.Installer.PostStepReply(ctx, &installer.PostStepReplyParams{
+				ClusterID: clusterID,
+				HostID:    *c.Hosts[0].ID,
+				Reply: &models.StepReply{
+					ExitCode: 1,
+					StepType: models.StepTypeInstall,
+					Output:   "blabla",
+					Error:    "Some random error",
+					StepID:   string(models.StepTypeInstall),
+				},
+			})
+			Expect(err).ShouldNot(HaveOccurred())
+
+			By("Verifying installation failed")
+			waitForClusterState(ctx, clusterID, "error", defaultWaitForClusterStateTimeout, "failed")
+		})
+
+		It("[only_k8s]install_cluster assisted-installer already running", func() {
+			By("Installing cluster till finalize")
+			c := installCluster(clusterID)
+			Expect(swag.StringValue(c.Status)).Should(Equal("installing"))
+			Expect(swag.StringValue(c.StatusInfo)).Should(Equal("Installation in progress"))
+			Expect(len(c.Hosts)).Should(Equal(4))
+			for _, host := range c.Hosts {
+				Expect(swag.StringValue(host.Status)).Should(Equal("installing"))
+			}
+
+			for _, host := range c.Hosts {
+				updateProgress(*host.ID, clusterID, models.HostStageDone)
+			}
+			// post failure to execute the install command due to a running assisted-installer
+			_, err := agentBMClient.Installer.PostStepReply(ctx, &installer.PostStepReplyParams{
+				ClusterID: clusterID,
+				HostID:    *c.Hosts[0].ID,
+				Reply: &models.StepReply{
+					ExitCode: 137,
```

Comment:
please add a comment about this specific error (i see the comment above but it's not clear why 137)

Reply:
Having a constant variable would make it clearer :)